inst/templates/derivationchain/custom.js

```diff
@@ -368,8 +368,15 @@ function generateLegend(nodes) {
   var visibleProcessors = [];
   var addExternalLine = false;
   var legendContainer = $("#legendContainer");
-  $(legendContainer).html('<strong>Explanation</strong><br/><br/><span style="text-decoration:underline">Processor Types:</span> <br/>');
-
+  $(legendContainer).html('<strong>Explanation</strong><br/><br/><span style="text-decoration:underline">Dataset Processor Types:</span> <br/>');
+  
+  var flags = [], output = [], l = nodes.length, i;
```

Comment:
this loops through the node locations and if there are different locations in the list, add to another array which we'll check later before adding that dashed line to the legend